go/private/rules/test.bzl

```diff
@@ -99,6 +102,10 @@ def _go_test_impl(ctx):
     external_archive = go.archive(go, external_go_info, is_external_pkg = True)
     if external_archive.data._validation_output:
         validation_outputs.append(external_archive.data._validation_output)
+    if external_archive.data._nogo_fix_output:
+        # internal vs external archive refers to the same package vs separate package.
+        # we include the nogo fixes for transitive dependency too.
```

Comment:
What does “transitive dependency" mean here? Aren't we just reporting no-go fixes for both sources of internal and of external tests?

Reply:
Removed the comment. The code is self-explanatory enough